unittests/test_schedulers.py

```diff
@@ -784,12 +771,6 @@ def test_flex_alloc_valid_multiple_constraints(make_flexible_job):
     assert job.num_tasks == 4
 
 
-def test_flex_alloc_valid_partition_cmd(make_flexible_job):
-    job = make_flexible_job('all', sched_partition='p2')
-    prepare_job(job)
-    assert job.num_tasks == 8
```

Comment:
You need this test, but pass the partition as a `job.option`. These set of tests exercise a complete different aspect than that the options are passed correctly to the job. So *all* the tests should remain and be adapted.